src/ClientData/CaptureData.cpp

```diff
@@ -270,6 +270,17 @@ const std::vector<uint64_t>* CaptureData::GetSortedTimerDurationsForScopeId(
   return all_scopes_->GetSortedTimerDurationsForScopeId(scope_id);
 }
 
+std::shared_ptr<ScopeStatsCollection> CaptureData::GetAllScopeStatsCollection() const {
+  return all_scopes_;
```

Comment:
Yeah talked to beckerhe offline about it; it was so that the LiveTab and CaptureData could share the information when nothing was selected, but after discussing I agree it shouldn't be shared. Updated in next commit